web/public/mgmt/config.yml

```diff
@@ -2995,6 +2995,32 @@ collections:
       - { label: "Internal Notes", name: "notesMd", widget: "markdown", required: false }
       - { label: "Filename (disabled after file creation)", name: "filename", widget: "write-once-read-only" }
       - { label: "Name/Title", name: "name", widget: "string" }
+      - {
+          label: "Apply to all - overrides industry and sector dropdown",
```

Comment:
[dust] Capitalize some of these labels: change to `"Apply to All - Overrides Industry and Sector Dropdown"`

Reply:
Hey Anju, I looked into the CMS and noticed all the characters are capitalized in the CMS; based on that, I'm going to leave it as it.

Reply:
<img width="151" alt="Screenshot 2024-01-17 at 9 17 45 AM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/ae6ccaea-5df6-41d9-a0d3-4452fa99d41d">